src/pybind/mgr/dashboard/module.py

```diff
@@ -66,7 +66,6 @@ def fixed_init(hc_self, server, sock, makefile=CP_fileobject):
                       handle_option_command
 
 from .plugins import PLUGIN_MANAGER
-from .plugins import feature_toggles  # noqa # pylint: disable=unused-import
```

Comment:
The ``disable=unused-import`` makes me wonder if this import is necessary here.

Reply:
If you want to import something, but you only need to execute the module once (e.g. you're only registering callbacks), pyling complains at this point. 

`disable=unused-import` marks the import as valid despite of not being used.

Reply:
That's why i was asking. Reverted the removal.